main_full.py

```
@@ -0,0 +1,538 @@
+            await callback.message.answer(T["start"], reply_markup=kb_main())
+        await callback.answer()
+
+    @dp.callback_query(F.data == "show_info")
+    async def cb_show_info(callback: CallbackQuery, state: FSMContext):
+        await state.clear()
+        from src.utils.file_cache import FileCache
+        info_cache = FileCache("connection_info.txt")
+        text = await info_cache.get()
+        
+        if not text:
+            text = "📋 Информация о подключении:\n\n" + \
```

Ruff (0.12.2)

String contains ambiguous `о` (CYRILLIC SMALL LETTER O). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)